library/src/test/java/ai/promoted/FakeAppRuntimeEnvironment.kt

```diff
@@ -0,0 +1,13 @@
+package ai.promoted
+
+import ai.promoted.platform.AppRuntimeEnvironment
+import android.content.Context
+
+data class FakeAppRuntimeEnvironment(
+    override val isDebuggable: Boolean
+) : AppRuntimeEnvironment {
+    @Suppress("EmptyFunctionBlock")
+    override fun initialize(context: Context) {
+
```

Comment:
Is this a no-op on purpose?

Reply:
Yes this is for unit testing / mocking the runtime environment on the phone